src/components/modals/AddTokenGatedRoomModal.tsx

```diff
@@ -151,11 +151,11 @@ export default function AddTokenGatedRoomModal({
                 )}
                 {!!hasIds && (
                     <>
-                        <Label htmlFor="tokenId">Token ID</Label>
+                        <Label htmlFor="tokenId">Token IDs (comma separated)</Label>
                         <TextField
                             id="tokenId"
-                            value={tokenId}
-                            onChange={({ target }) => void setTokenId(target.value)}
+                            value={tokenIds.join(',')}
```

Comment:
This operation is going to make the input act strange. Especially the selection (caret) in a multi-id scenario (separate ids with something more than a comma, e.g. `,<space>`).

Ideally `onChange` updates the state and the state is used directly (as is) in the `value` prop.

One way to fix it: drop white spaces on submission, not while entering ids.